app/models/allocation_decision.rb

```diff
@@ -1,21 +1,19 @@
 # frozen_string_literal: true
 
 class AllocationDecision < ApplicationRecord
-  DESCRIPTIONS = ["Offence triable only by court martial",
-                  "Indictable only offence",
-                  "No mode of Trial Either way offence",
-                  "Defendant consents to summary trial",
-                  "Commanding Officer orders trial by court martial",
-                  "Defendant chooses trial by court martial 1",
-                  "Defendant elects trial by jury",
-                  "Summary only offence",
-                  "Youth offence triable on indictment only (homicide relevant firearms dangerousness)",
-                  "Youth Court directs trial by jury (grave crime)",
-                  "Low value offence triable summarily only",
-                  "Indictable only (previous convictions relevant firearms offence)",
-                  "Court directs trial by jury"].freeze
-
-  CODES = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].freeze
+  DESCRIPTIONS = { "Offence triable only by court martial" => 8,
+                   "Indictable-only offence" => 2,
+                   "No mode of Trial - Either way offence" => 7,
+                   "Defendant consents to summary trial" => 3,
+                   "Commanding Officer orders trial by court martial" => 10,
+                   "Defendant chooses trial by court martial 1" => 9,
+                   "Defendant elects trial by jury" => 4,
+                   "Summary-only offence" => 1,
+                   "Youth - offence triable on indictment only (homicide / relevant firearms dangerousness)" => 2,
```

Comment:
This one is:
`"Youth - offence triable on indictment only (homicide / relevant firearms / dangerousness)" => 2,`